api/envoy/extensions/upstreams/http/v3/http_protocol_options.proto

```diff
@@ -140,7 +140,6 @@ message HttpProtocolOptions {
     option (validate.required) = true;
 
     // To explicitly configure either HTTP/1 or HTTP/2 (but not both!) use ``explicit_http_config``.
-    // If the ``explicit_http_config`` is empty, HTTP/1.1 is used.
```

Comment:
is this related to the change?  If not, we can remove?

Reply:
Yes. This is unrelated.  But seems the checks failed if I don't keep this change.

Reply:
Removed again.